tools/extensions/old_extension_db.json

```diff
@@ -0,0 +1 @@
t_to_untrusted_downstream_and_upstream", "undocumented": false, "status": "stable", "categories": ["envoy.internal_redirect_predicates"]}, "envoy.internal_redirect_predicates.safe_cross_scheme": {"security_posture": "robust_to_untrusted_downstream_and_upstream", "undocumented": false, "status": "stable", "categories": ["envoy.internal_redirect_predicates"]}, "envoy.upstreams.http.http": {"security_posture": "robust_to_untrusted_downstream", "undocumented": false, "status": "stable", "categories": ["envoy.upstreams"]}, "envoy.upstreams.http.tcp": {"security_posture": "robust_to_untrusted_downstream", "undocumented": false, "status": "stable", "categories": ["envoy.upstreams"]}, "envoy.watchdog.profile_action": {"security_posture": "data_plane_agnostic", "undocumented": false, "status": "alpha", "categories": ["envoy.guarddog_actions"]}, "envoy.wasm.runtime.null": {"security_posture": "unknown", "undocumented": false, "status": "alpha", "categories": ["envoy.wasm.runtime"]}, "envoy.wasm.runtime.v8": {"security_posture": "unknown", "undocumented": false, "status": "alpha", "categories": ["envoy.wasm.runtime"]}, "envoy.wasm.runtime.wavm": {"security_posture": "unknown", "undocumented": false, "status": "alpha", "categories": ["envoy.wasm.runtime"]}, "envoy.wasm.runtime.wasmtime": {"security_posture": "unknown", "undocumented": false, "status": "alpha", "categories": ["envoy.wasm.runtime"]}, "envoy.rate_limit_descriptors.expr": {"security_posture": "unknown", "undocumented": false, "status": "stable", "categories": ["envoy.rate_limit_descriptors"]}, "envoy.io_socket.user_space": {"security_posture": "unknown", "undocumented": true, "status": "wip", "categories": ["envoy.io_socket"]}, "envoy.tls.cert_validator.spiffe": {"security_posture": "unknown", "undocumented": false, "status": "wip", "categories": ["envoy.tls.cert_validator"]}, "envoy.http.stateful_header_formatters.preserve_case": {"security_posture": "robust_to_untrusted_downstream_and_upstream", "undocumented": false, "status": "stable", "categories": ["envoy.http.stateful_header_formatters"]}, "envoy.transport_sockets.tls": {"security_posture": "robust_to_untrusted_downstream_and_upstream", "undocumented": false, "status": "stable", "categories": ["envoy.transport_sockets.downstream", "envoy.transport_sockets.upstream"]}, "envoy.upstreams.http.generic": {"security_posture": "robust_to_untrusted_downstream", "undocumented": false, "status": "stable", "categories": ["envoy.upstreams"]}, "envoy.upstreams.tcp.generic": {"security_posture": "robust_to_untrusted_downstream", "undocumented": false, "status": "stable", "categories": ["envoy.upstreams"]}, "envoy.upstreams.http.http_protocol_options": {"security_posture": "robust_to_untrusted_downstream", "undocumented": false, "status": "stable", "categories": ["envoy.upstreams"]}, "envoy.request_id.uuid": {"security_posture": "robust_to_untrusted_downstream_and_upstream", "undocumented": false, "status": "stable", "categories": ["envoy.request_id"]}}
```

Comment:
this has been added to prevent regression - should be removed before landing this PR